service/service_ontology.ttl

```diff
@@ -0,0 +1,23 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix service: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/service/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+
+service: a owl:Ontology ;
+    rdfs:label "ontology definition for a simulation service"@en ;
+    dcterms:contributor "Valentin Fischer (TT)" ;
+    owl:versionInfo "0.1" .
+
+service:Service a owl:Class ;

```

Comment:
Muss Service nicht Subclass von gx:SoftwareResource sein?

Reply:
Das hatte ich ursprünglich so, allerdings scheint mir, dass eine SoftwareResource dann benutzt wird, wenn ich eine Software habe, welche in an den Consumer schicke. Allerdings führe ich ja eine Software bei mir aus und schicke dem Consumer dann Daten zurück, also passt das meiner Ansicht nach nicht wirklich. Ich kann das aber auch gerne anpassen wenn du das für sinnvoll hältst :)

Reply:
Wir brauchen definitiv eine Superklasse, die von einer Gaia-X-Basisklasse ableitet (`Resource` oder `ServiceOffering`).
Bisher haben wir Services als `SoftwareResource` angesehen (ändert sich vermutlich mit dem Loire Release).
Was aber auch möglich wäre sind
- `Image` (leitet von `SoftwareResouce` ab
- `Compute` (leitet von `Infrastructure` ab, was wiederum ein `ServiceOffering` ist.

Der Einfachheit halber würde ich erstmal auf `SoftwareResource` gehen, weil das eines der nächsten Sachen ist, die wir im Portal angehen wollen.

Reply:
Okay, wurde angepasst in dem neuesten commit.